docs/osquery/osquery.asciidoc

```diff
@@ -121,9 +121,12 @@ image::images/scheduled-pack.png[Shows queries in the pack and details about eac
 
 [float]
 [[osquery-manage-query]]
-== Edit saved queries
+== Saved queries
 
-Add or edit saved queries from the *Saved queries* tab.
+You can save queries after running a live query or from the *Saved queries* tab. 
+Once you save a query, you can only edit it from the *Saved queries* area.
+
+How to add or edit saved queries from the *Saved queries* tab:
 
 . Go to the saved queries, then click **Add saved query** or the edit icon.
```

Comment:
If "saved queries" is referring to the Saved queries tab, then put in bold and match the text in the UI:

> Go to **Saved queries**,  and then click **Add saved query** or the edit icon.